doc/content/integrations/aws-iot/default/architecture.md

```diff
@@ -24,6 +24,10 @@ This is a serverless deployment: there are no compute resources being deployed.
 
 This integration supports true LoRaWAN end-to-end encryption: the application payload is encrypted on the end device with the LoRaWAN AppSKey, and decrypted in your AWS Account. The underlying network infrastructure passes your application payload in the encrypted form - it cannot see your data.
 
-When enabled, this integration configures Join Server with your key encryption key (KEK) that is generated in your AWS Account and stored as a secret in [Secrets Manager](https://aws.amazon.com/secrets-manager/). Join Server encrypts the AppSKey with the KEK before passing it to the network layer. The network layer sends the encrypted AppSKey to your AWS Account, where it gets decrypted. This feature only works with devices registered in Join Server.
+When end-to-end encryption is enabled, this integration configures Join Server with your key encryption key (KEK) that is generated in your AWS Account and stored as a secret in [Secrets Manager](https://aws.amazon.com/secrets-manager/). Join Server encrypts the AppSKey with the KEK before passing it to the network layer. The network layer sends the encrypted AppSKey to your AWS Account, where it gets decrypted. 
+
+{{< note >}} This feature only works with devices registered in Join Server.{{</ note >}}
```

Comment:
This note is a bit unclear. It works both with the in-cluster Join Server and The Things Join Server.

This whole end-to-end encryption only works with OTAA devices. Is that what you meant?

Reply:
This was already here actually, I just wrapped it as a note because it seemed important. In which case it wouldn't work, only for ABP devices or?

Reply:
Ah ok. If it's an old text, then it's obsolete, as this used to work only with The Things Join Server, and now also with in-cluster Join Server. I think it's best to just remove this note.